src/luarocks/fs/unix/tools.lua

```diff
@@ -171,7 +171,7 @@ end
 -- If not given, name is derived from input file.
 -- @return boolean: true on success; nil and error message on failure.
 function tools.gunzip(infile, outfile)
-   return uncompress("gz", "gunzip", infile, outfile)
+   return uncompress("gz", "gzip", "-d", infile, outfile)
```

Comment:
We can't use `gunzip` here on MSYS2.
`gunzip` on MSYS2 implemented by shell script.
It's not called from `cmd.exe`.